site/.eslintrc.json

```diff
@@ -0,0 +1,29 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:import/recommended",
+    "plugin:prettier/recommended"
+  ],
+  "settings": {
+    "react": {
+      "version": "detect"
```

Comment:
> Warning: React version was set to "detect" in eslint-plugin-react settings, but the "react" package is not installed. Assuming latest React version for linting.

Just an FYI in case you haven't seen it, there is a warning in console on linting if `yarn install` is only run in the root of the repo. I don't mind leaving it as is though, since it has a fallback anyway. So functionally it has zero impact

Reply:
Oh, I see. It's the first time I realize CI has also been linting the website all this time...

Reply:
:smile: